{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--0a706b07-2f47-4e14-857e-4a307ca46181",
    "created": "2020-05-26T20:29:41.011034Z",
    "modified": "2020-05-26T20:29:41.011034Z",
    "name": "CVE-2017-9163",
    "description": "libautotrace.a in AutoTrace 0.31.1 has a \"\"cannot be represented in type int\"\" issue in pxl-outline.c:106:54."
}